src/accelerate/commands/config/cluster.py

```diff
@@ -50,6 +50,25 @@ def get_cluster_input():
                 "What is the port you will use to communicate with the main process? ",
                 lambda x: int(x),
             )
+
+        # TODO: add other deepspeed args
+        if distributed_type == DistributedType.DEEPSPEED:
+            zero_stage = _ask_field(
+                "What should be your DeepSpeed's ZeRO optimization stage (0, 1, 2)? [2]: ",
+                lambda x: int(x),
+                default=2,
+            )
+            gradient_accumulation_steps = _ask_field(
+                "What should be your number of gradient accumulation steps? [1]: ",
+                lambda x: int(x),
+                default=1,
+            )
+            is_train_batch_min = _ask_field(
+                "Should accelerate choose minimum batch size incase both train & eval dataloaders are passed? [True]: ",
```

Comment:
I think this one should just be left at True: there are going to be tons of parameters for deepspeed so let's keep the essential ones in the question (and use the default for the others). Users can edit the config after!